src/main/java/com/shaft/cucumber/ElementSteps.java

```diff
@@ -18,27 +17,27 @@ public ElementSteps(ThreadLocal<WebDriver> driver) {
         }
     }
 
-    protected static By getLocatorFromTypeAndValue(LocatorType type, String locatorValue) {
-        switch (type) {
-            case ID -> {
+    protected static By getLocatorFromTypeAndValue(String locatorType, String locatorValue) {
```

Comment:
Codacy found an issue: [The method 'getLocatorFromTypeAndValue(String, String)' has an NPath complexity of 2147483647](https://app.codacy.com/manual/mohab.mohieeldeen/SHAFT_ENGINE/pullRequest?prid=5909418)